src/styles/_common.scss

```diff
@@ -419,6 +419,34 @@ i {
       }
       overflow: hidden;
     }
+    .twitter-feed {
```

Comment:
Perhaps `.twitter-post` would be more appropriate?
And where is the responsive styling? Did you test this on multiple screens sizes?